Documentation/Tools/HolographicRemoting.md

```diff
@@ -47,9 +38,7 @@ These issues are particularly relevant when running on **Unity 2019.3** or later
 #### MSBuildForUnity package import via writing into the package.manifest
 
 > [!Note]
-> There is a known issue that prevents MSBuild for Unity from functioning properly on some versions of Unity 2019. To avoid this issue, the MRTK does not support MSBuild for Unity on Unity 2019.3.
->
-> To acquire the required NuGet package when running on Unity 2019.3, please refer to the [manual installation instructions](#manual-dotnetadapter-installation).
+> There is an issue that prevents MSBuild for Unity from functioning properly on some versions of Unity 2019. If **Mixed Reality Toolkit** > **MSBuild** > **Use MSBuild for Unity dependency resolution** does not work correctly, please refer to the [manual installation instructions](#manual-dotnetadapter-installation).
```

Comment:
Should this just be "on Unity 2019" (i.e. we expect that people on the latest versions to be affected by this, and shoudl be upfront and maybe not give people hope that later versions would be resolved because usually I would expect later versions to have fixed issues rather than be experiencing them)

Reply:
it has failed on earlier versions and has been stable on current versions. not sure the best wording here.